src/server/app/admin/services/users.authentication.server.service.js

```diff
@@ -44,6 +46,46 @@ module.exports.initializeNewUser = function(user) {
 	return q(user);
 };
 
+// Send email alert about new account request
+module.exports.signupEmail = function(user, req) {
+	let defer = q.defer();
+
+	req.app.render('../../admin/templates/user-signup-alert-email', {
+		name: user.name,
+		username: user.username,
+		appName: config.app.title,
+		url: `${config.app.baseUrl}/#/admin/users`
```

Comment:
Good call.  Depending on which one gets merged first I will make the change in the other.